src/components/molecules/Settings/Workspace/Asset/AssetDeleteModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
```

Comment:
Can we move this component into the AssetModal folder in Common? Since it is used in the AssetContainer and I can't see it being used elsewhere, it would make more sense to keep it together with the AssetModal